contribute/release-process.md

```diff
@@ -219,8 +219,14 @@ Run the following commands:
 
 ```shell
 cd $PULSAR_HOME/docker
-./build.sh
-DOCKER_USER=<your-username> DOCKER_PASSWORD=<your-password> DOCKER_ORG=<your-username> ./publish.sh
+DOCKER_USER=<your-username>
+docker login $DOCKER_USER -u $DOCKER_USER -p <your-password>
+mvn install -DUBUNTU_MIRROR=http://azure.archive.ubuntu.com/ubuntu/ \
+    -DskipTests \
+    -Pmain,docker -Pdocker-push \
+    -Ddocker.platforms=linux/amd64,linux/arm64 \
+    -Ddocker.organization=$DOCKER_USER \
```

Comment:
Since the variable `DOCKER_USER` might not be the same as `docker.organization`, it is better to change `$DOCKER_USER` to `$DOCKER_ORGANIZATION` here